rotkehlchen/tests/utils/globaldb.py

```diff
@@ -107,3 +110,20 @@ def patch_for_globaldb_migrations(stack: ExitStack, new_list: list) -> ExitStack
         ),
     )
     return stack
+
+
+def globaldb_get_general_cache_last_queried_ts(
+        cursor: DBCursor,
+        key_parts: Iterable[Union[str, GeneralCacheType]],
+        value: str,
+) -> Optional[Timestamp]:
+    """Function to get timestamp at which pair key - value was queried last time."""
+    cache_key, _ = compute_cache_key(key_parts)
+    cursor.execute(
+        'SELECT last_queried_ts FROM general_cache WHERE key=? AND value=?',
```

Comment:
>  But i guess key_value is always unique so it will always return a since last_queried time stamp, isnt it?

This function queries data from `general_cache` table, which contains data with multiple values per key, doesn't it?

Reply:
+1 on Alexey's question here

Reply:
The thing is, I did not write that function. I was just adviced by Yabir to Carry it from `cache.py` to a `test util` since it is only used in testing. The function existed already and was written as such. but i could implement as you ask.